sdk/networkcloud/Azure.ResourceManager.NetworkCloud/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Please specify the expected release date by replacing the `Unreleased`, the format is yyyy-MM-dd

Reply:
I updated Microsoft.NetworkCloud